sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/util/TestKitHelper.java

```diff
@@ -0,0 +1,25 @@
+package de.ikor.sip.foundation.testkit.util;
+
+import de.ikor.sip.foundation.core.util.SIPExchangeHelper;
+import de.ikor.sip.foundation.testkit.configurationproperties.models.MessageProperties;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.support.MessageHelper;
+
+/** Utility class that changes the {@link Exchange} */
+@Slf4j
+public class TestKitHelper extends SIPExchangeHelper {
```

Comment:
I don't think we need this class.
It could be a constructor of MessageProperties or static instantiate method

Reply:
The class is now needed, since it has additional functionalities

Reply:
still, it feels like mapToMessageProperties better fits as MessageProperties constructor to mee